app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if not visitor["vaccine"]:
+                raise NotVaccinatedError("NotVaccinatedError")
+        except KeyError:
+            raise NotVaccinatedError("NotVaccinatedError")
+        try:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError("OutdatedVaccineError")
+            if visitor["wearing_a_mask"] is False:
+                raise NotWearingMaskError("NotWearingMaskError")

```

Comment:
The error message 'NotWearingMaskError' is not descriptive. According to the checklist, you should provide a more descriptive message when raising exceptions. Consider including more context about the error.